### DISCUSSION TITLE
文化幣用途

### DISCUSSION THREAD

**Discussion by alvin119** [1101-GS4719-C/Python-Community-GS4719-C#1838] [General]
1/20文化部又會給年齡介於16~22歲的學生再發一次文化幣，總共1200點，
好奇各位會拿這1200點拿去做些什麼事，或是上一次有拿去從事什麼活動?
自己上一次是拿去買某個鋼琴音樂會的門票，結果還要貼錢...

**Comment by B1nn9** [comment]
上次的原本想拿去吃春水堂，結果沒注意到他公布停止使用，最後就只能想辦法變現了😫

**Comment by 110101501** [comment]
自己是有拿文化幣去聽音樂會，而且當時還有回饋，所以其實蠻划算的。
有聽聞過身邊的朋友，有人是買教科書、有人則是做首手作、工藝品，其實功能還蠻多元，主要就取決於自己想怎麼使用。

**Comment by happy246810** [comment]
我是拿去買書，因為原文書真的好貴，而且用文化幣買，即使你讀完了，你賣掉書，你還可以賺到錢，我覺得這也是一個另類的兌現方法。這樣不只賺到錢，還可以解決你的需求，我覺得超棒。

**Comment by YOGA012** [comment]
我都拿去買漫畫，這邊推一個位於高雄市新興區球庭路91號的大鵬漫畫小說量販，因為是獨立書局，所以文化幣會變成1.5倍，1800元可以買一堆想收的漫畫，他們也有臉書粉專[https://www.facebook.com/ta.penh.bookstore](url)，小編還蠻幽默的。

**Comment by jeff110303549** [comment]
我花在一個製作手工地毯的課程上，因為我從美國回來的表姐剛好回到台灣，而她看到了那個課程就提出要和我一起去嘗試看看，尤其是他還跟我說這個課程在美國的花費非常高，更讓我覺得文化比花的值得
